static/.htaccess

```
@@ -6,6 +6,11 @@ RewriteRule (.*) https://%{SERVER_NAME}/$1 [R=301,L]
 # Disable directory listing and multi views
 Options -Indexes -MultiViews
 
+# Rename Console SM to Administration API (Self-Managed)
+RewriteRule ^docs/apis-tools/console-sm-api/?$ /docs/apis-tools/administration-sm-api/$1 [R=301,L]
+RewriteRule ^docs/apis-tools/console-sm-api/console-sm-api-authentication/?$ /docs/apis-tools/administration-sm-api/administration-sm-api-authentication/$1 [R=301,L]
+RewriteRule ^docs/apis-tools/console-sm-api/console-sm-api-overview/?$ /docs/apis-tools/administration-sm-api/administration-sm-api-overview/$1 [R=301,L]
```

```suggestion
RewriteRule ^docs/apis-tools/console-sm-api/(.*)$ /docs/apis-tools/administration-sm-api/$1 [R=301,L]
```

This one rule will catch all URLs within the `console-sm-api` path, and redirect them to the same thing except `administration-sm-api-overview`. Which means it will take care of the three you've individually called out here, as well as all the generated pages within the `specifications` folder.